packages/core/src/components/way-input/way-input.scss

```diff
@@ -0,0 +1,97 @@
+:host {
+  /**
+    * @prop --border-radius: Border radius of the input
+    * @prop --border-width: Border width of the button
+    * @prop --border-style: Border style of the button
+    * @prop --border-color: Border color of the button
+    */
+  display: inline-block;
+
+  width: auto;
+
+  font-family: var(--way-font-family, inherit);
+  font-size: var(--way-font-size);
+  line-height: 1.25rem;
+
+  user-select: none;
+  vertical-align: top;
+  vertical-align: -webkit-baseline-middle;
+  pointer-events: auto;
+
+  font-kerning: none;
+
+  --border-radius: var(--way-form-element-border-radius-medium);
+  --border-width: var(--way-form-element-input-border-width);
+  --border-color: var(--way-color-medium-tint);
+  --border-color-focus: var(--way-color-light);
+  --border-style: solid;
+  --font-size: var(--way-form-element-font-size-small);
```

Comment:
**issue**: waarom zo kleine font-sizes? In het design zijn ze gewoon de font-size van body tekst. Dan is die x-small variant ook niet nodig.